root/defaults/nginx/authelia-server.conf.sample

```diff
@@ -1,4 +1,4 @@
-## Version 2023/04/27 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-server.conf.sample
+## Version 2024/03/14 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-server.conf.sample
 # Make sure that your authelia container is in the same user defined bridge network and is named authelia
 # Rename /config/nginx/proxy-confs/authelia.subdomain.conf.sample to /config/nginx/proxy-confs/authelia.subdomain.conf
 # Make sure that the authelia configuration.yml has 'path: "authelia"' defined
```

Comment:
add line 4 and 5 from authelia-location here too

Reply:
Resolved in https://github.com/linuxserver/docker-swag/pull/465/commits/c63b437e1f40acb19e7678cbc962d01c10879b5d